components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -140,8 +140,42 @@ __attribute__((objc_subclassing_restricted))
  */
 @property(nonatomic, nullable, strong) UIFont *actionFont;
 
+/**
+ The color applied to the sheet view of the action sheet controller.
+ */
 @property(nonatomic, nonnull, strong) UIColor *backgroundColor;
 
+/**
+ The color applied that will be applied to message if there is a message. If there is no message
+ then this color will be applied to the title.
+ */
+@property(nonatomic, strong, nullable) UIColor *primaryTextColor;
+
+/**
+ The color applied to the title if there is both a title and a message.
+ */
+@property(nonatomic, strong, nullable) UIColor *secondaryTextColor;
+
+/**
+ The color applied to the title of the action sheet controller.
+ */
+@property(nonatomic, strong, nullable) UIColor *titleTextColor;
+
+/**
+ The color applied to the message of the actio sheet controller.
+ */
+@property(nonatomic, strong, nullable) UIColor *messageTextColor;
+
+/**
+ The color applied to the item list labels of action sheet controller.
+ */
+@property(nonatomic, strong, nullable) UIColor *actionTextColor;
+
+/**
+ The color applied to the icon items of action sheet controller.
+ */
+@property(nonatomic, strong, nullable) UIColor *actionTintColor;
```

Comment:
Why not use `UIView.tintColor`? That would match what our other components do. Are we using that for some other part of the UI?

Reply:
The tint color is being applied to the image. A client may need an image that is red and an label that is blue.

Reply:
I thought UILabel doesn't apply `tintColor` to the text color? At least I've never been able to get it to work.  Are you finding that `tintColor` is being applied to the label's text color in some circumstances?

Reply:
If you are referring to the implementation why don't I apply `tintColor`, I think there is a disconnect because that is what is currently being used. If you are asking why the _API_ isn't `@property(nonatomic, strong, nullable) UIColor *tintColor` I feel that that can be confusing because clients may wonder what all this `tint` will be applied to. If you are referencing the `UIKit` equivalent `UIAlertController` with style `ActionSheet` doesn't have a `tintColor`. For now, I feel this is a better `API`.

Reply:
There's some nuance here to ensure we consider:

1. If a client sets the tintColor of the action sheet, this will be propagated to any subviews that respond to tintColor. @codeman7 If we decide to go with this explicit actionTintColor API we need to make sure it plays nicely with the existing tintColor behavior.
2. I like the idea of having an explicit API that very clearly affects one thing, but 1. makes me a bit nervous that we might collide with expectations about how tintColor is propagating to our component.